Toolchain/nosman/src/nosman/command/install.rs

```diff
@@ -22,7 +22,7 @@ impl From<ZipError> for CommandError {
 }
 
 impl InstallCommand {
-    pub(crate) fn run_install(&self, workspace: &mut Workspace, package_name: &str, version_opt: Option<&String>, exact: bool, output_dir: &PathBuf, prefix: Option<&String>, fetch_index: bool) -> CommandResult {
+    pub(crate) fn run_install(&self, workspace: &mut Workspace, package_name: &str, version_opt: Option<&String>, exact: bool, output_dir: &PathBuf, prefix: Option<&String>, fetch_index: bool, no_dependency: bool) -> CommandResult {
```

Comment:
Since we now have three booleans for this method, we can combine it into `InstallFlags` enum. 

```rust

#[derive(Clone, Copy)]
pub struct InstallFlags(u8);
bitflags! {
    impl InstallFlags: u8 {
        const UpdatePackageIndex = 0b1;
        const WithoutDependencies = 0b10;
        const InstallExactVersion = 0b100;
    }
}
```